nodejs/eks/cluster.ts

```diff
@@ -422,6 +423,20 @@ export function createCore(name: string, args: ClusterOptions, parent: pulumi.Co
         }, { parent, provider });
     }
 
+    // Create the cluster encryption provider for using envelope encryption on
+    // Kubernetes secrets.
+    let encryptionProvider: pulumi.Output<aws.types.input.eks.ClusterEncryptionConfigProvider> | undefined;
+    let encryptionConfig: pulumi.Output<aws.types.input.eks.ClusterEncryptionConfig> | undefined;
+    if (args.encryptionConfigKeyArn) {
+        encryptionProvider = pulumi.output(args.encryptionConfigKeyArn).apply(
+            keyArn => <aws.types.input.eks.ClusterEncryptionConfigProvider>({keyArn}),
+        );
+        encryptionConfig = encryptionProvider.apply(encryptProvider => (<aws.types.input.eks.ClusterEncryptionConfig>{
+            provider: encryptionProvider,
```

Comment:
Should this be `encryptProvider`? Looks like the apply arg isn't used.

Reply:
Good catch 👍